src/librustc_codegen_llvm/abi.rs

```diff
@@ -476,6 +476,15 @@ impl<'a, 'tcx> FnTypeExt<'a, 'tcx> for FnType<'tcx, Ty<'tcx>> {
                 }
             }
 
+            // HACK(nox): Pointee alignment for indirect sret arguments make
+            // LLVM unhappy at least on i686-pc-windows, and that leads to some
+            // tests failing (namely extern-pass-TwoU64s and struct-return).
```

Comment:
I don't thinks this is well-placed, since it runs *before* the `adjust_for_cabi` call in `adjust_for_abi`.

Reply:
Where should it go, then?